cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -167,9 +167,10 @@
 
   <property>
     <name>zookeeper.quorum</name>
-    <value>127.0.0.1:2181/${root.namespace}</value>
+    <value>FQDN1:2181,FQDN2:2181/${root.namespace}</value>
```

Comment:
This changes our default config from one that might work on a dev setup, to one that is always invalid (and possibly java might think it's malformed?).  I don't think we should do this.  These "substitute-me" tokens are only appropriate for example files such as https://github.com/caskdata/cdap/blob/develop/cdap-distributions/src/etc/cdap/conf.dist/cdap-site.xml.example.  Also update description accordingly.